code401Challenges/src/main/java/code401Challenges/GetEdge/GetEdge.java

```diff
@@ -0,0 +1,67 @@
+package code401Challenges.GetEdge;
+
+import code401Challenges.Graph.Edge;
+import code401Challenges.Graph.Graph;
+import code401Challenges.Graph.Node;
+
+import java.util.HashSet;
+
+public class GetEdge<T> {
+
+    boolean output = false;
+    int cost = 0;
+
+    public GetEdge(){}
```

Comment:
You don't need a no-args constructor unless you're working with Spring and a database